src/dataviews/dataview-model-base.js

```diff
@@ -185,6 +185,7 @@ module.exports = Model.extend({
   // Class props
   {
     ATTRS_NAMES: [
+      'type',
```

Comment:
IDs are not important in windshaft, if they can be used in JSON keys it will do OK. But it probably makes a lot of sense to, as you are proposing, to use the IDs from the editor so it's easier to trace up to the origin.